### PR TITLE
Add pure Kotlin rss standard parser.

### PR DESCRIPTION
**Pull request by ivanisidrowu**
- Made 'testCommon' a pure Kotlin module.
- Moved common classes to the `kotlin` module.
    - `ChannelItemTestData`: From module `android` to `testCommon`.
- Fixed tests.
    - `DatabaseRssCacheLocalTest`: applicationContext is not initiated.
- Added pure Kotlin parser `RssStandardParser` in `kotlin` module.
    - Used JAVA DOMParser to parse XML. I tried `SAX` and `SAtX`, but they do not support getting value by specific tags. Even thought it consumes much more memory, but the kotlin module mostly for non-mobile environment. I think it's acceptable.
- Added pure Kotlin parser `RssStandardParser` tests.

### CODE REVIEW

File: testCommon/build.gradle [hunk 1]
```diff
@@ -14,45 +14,20 @@
...
-    implementation 'androidx.core:core-ktx:1.3.2'
-    implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
 
     implementation deps.kotlinxCoroutinesCore
-    implementation deps.kotlinxCoroutinesAndroid
-    implementation deps.okhttp
+    implementation "com.squareup.okio:okio:2.9.0"
```
Comment (dev-weiqi) [inline testCommon/build.gradle:28]: 這個是為了哪邊需要呢？

**Review by dev-weiqi — APPROVED** [review]
done

File: testCommon/build.gradle (hunk 1 above)
Comment (ivanisidrowu) [reply testCommon/build.gradle:28]: `XmlFileReader` 裡面有個 `readByteString` 的 extension 其實是從 okio 來的，但原本是 import okhttp

**Review by AmyPlace — APPROVED** [review]
LGTM

**Pull request merged**